buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/publish/PublishingPlugin.groovy

```diff
@@ -121,11 +131,40 @@ class PublishingPlugin implements Plugin<Project> {
                             publisher.decorate(sub, it)
                         }
                     }
+
                     publishAllToLocal.dependsOn "$sub.path:publishMavenAarPublicationToMavenLocal"
                     publishAllToBuildDir.dependsOn "$sub.path:publishMavenAarPublicationToBuildDirRepository"
 
+                    // Update the changed and all sets each time an artifact is published. This is
+                    // used to build the JSON file of affected targets.
+                    sub.tasks.getByName("publishMavenAarPublicationToBuildDirRepository").doLast {
+                        def artifact = "$sub.group:$sub.name:$sub.version-SNAPSHOT"
```

Comment:
this will not work with ktx libraries as it's name is just `ktx` instead of `sdk-name-ktx`. One way you could try and solve it is by getting `groupId` and `artifactId` from the `mavenAar` Publication object

Reply:
Good point. It should be fixed now.